lib/hackney/income/tenancy_prioritiser/universal_housing_criteria.rb

```diff
@@ -21,7 +21,7 @@ def self.for_tenancy(universal_housing_client, tenancy_ref)
               ) t
               WHERE row = 1
             )
-            DECLARE @RemainingTransactions INT = (SELECT COUNT(*) FROM [dbo].[rtrans] WITH (NOLOCK) WHERE tag_ref = @TenancyRef)
+            DECLARE @RemainingTransactions INT = (SELECT COUNT(*) FROM [dbo].[rtrans] WITH (NOLOCK) WHERE tag_ref = @TenancyRef AND post_date > DATEADD(year,-1,GETDATE()))
```

Comment:
Suggest: 
```suggestion
            DECLARE @RemainingTransactions INT = (SELECT COUNT(tag_ref) FROM [dbo].[rtrans] WITH (NOLOCK) WHERE tag_ref = @TenancyRef AND post_date > DATEADD(year,-1,GETDATE()))
```
As ```Count(*)``` isn't efficient.

Reply:
🚢

Reply:
🚢

Reply:
🚢

Reply:
🚢